### ISSUE TITLE
 AutoPSK | Use existing TLS PSK identity fails due to no_log: true zabbix 5.4

### ISSUE THREAD

**Issue by solick** [ansible-collections/community.zabbix#607]
##### SUMMARY
zabbix agent role fails to to no_log: true setting:

    AutoPSK | Use existing TLS PSK identity
    FAILED! => {"censored": "the output has been hidden due to the fact that 'no_log: true' was specified for this result"}

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
zabbix_agent role

##### ANSIBLE VERSION

ansible [core 2.12.1]
  
  configured module search path = ['/Users/XXX/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/Cellar/ansible/5.1.0/libexec/lib/python3.10/site-packages/ansible
  ansible collection location = /Users/XXX/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/local/bin/ansible
  python version = 3.10.1 (main, Dec  6 2021, 23:20:29) [Clang 13.0.0 (clang-1300.0.29.3)]
  jinja version = 3.0.3
  libyaml = True

```

##### CONFIGURATION
<!--- Paste verbatim output from "ansible-config dump --only-changed" between quotes -->
DEFAULT_HOST_LIST(/Users/XXX/Development/Ansible/lmc-ansible/ansible.cfg) = ['/Users/lynmatten/Development/Ansible/lmc-ansible/hosts']
DEFAULT_REMOTE_USER(/Users/XXX/Development/Ansible/lmc-ansible/ansible.cfg) = root
DEFAULT_VAULT_PASSWORD_FILE(/Users/XXX/XXX/Ansible/XXX/ansible.cfg) = /Users/XXX/XXX/Ansible/XXX-vault-password

```

##### OS / ENVIRONMENT / Zabbix Version

Ubuntu 20.04
Zabbix 5.4

##### STEPS TO REPRODUCE

have set up the devices with role version 1.4.0, update to 1.5.1 and just using the agent role to run a new or existing zabbix agent

##### EXPECTED RESULTS

the role should run without error

##### ACTUAL RESULTS

AutoPSK | Use existing TLS PSK identity] 
FAILED! => {"censored": "the output has been hidden due to the fact that 'no_log: true' was specified for this result"}

```paste below

```

**Comment by dj-wasabi** [comment]
It has nothing to do with the `no_log: true`, as it just prevents the error to be displayed. Can you run the playbook command with `-vvvv`? Then it provides the error on why it is failing.

**Comment by ragingpastry** [comment]
This is resolved in master but not in 1.5.1. 

If you install this role from this git repository it should work

requirements.yml file
```yaml
---
collections:
  - name: git+https://github.com/ansible-collections/community.zabbix.git
    type: git
```

**Comment by solick** [comment]
I can confirm the the error does not appear with current master. THanks
